cryptarchia/test_fork_choice.py

```diff
@@ -8,26 +8,31 @@
 
 
 def make_block(parent_id: Id, slot: Slot, block_id: Id) -> BlockHeader:
```

Comment:
rename block_id to content_id now that we are using it for content?

Reply:
also, I think it would clean up the tests a bit if you have `make_block()` take `content` as input instead of `content_id`. Then here you can hash the content to get the content id for the header.